data/strings/types_strings.txt

```diff
@@ -7115,43 +7115,47 @@
     zh-Hant = 車庫
 
   [type.building.train_station]
-    en = Train Station
-    af = Treinstasie
-    ar = محطة قطار
-    az = Dəmiryolu stansiyası
-    cs = Železniční stanice
-    da = Togstation
-    de = Bahnhof
-    el = Σιδηροδρομικός σταθμός
-    es = Estación de tren
-    et = Rongijaam
-    eu = Tren geltokian
-    fa = ایستگاه قطار
-    fi = Rautatieasema
-    fr = Gare ferroviaire
-    he = תחנת רכבת
-    hi = रेलवे स्टेशन
-    hu = Vasútállomás
-    id = Rel kereta api
-    it = Stazione ferroviaria
-    ja = 鉄道駅
-    ko = 기차역
-    mr = रेल्वे स्थानक
-    nb = Togstasjon
-    nl = Treinstation
-    pl = Dworzec kolejowy
-    pt = Estação de comboios
-    pt-BR = Estação de trem
-    ro = Feroviar
-    ru = Ж/д вокзал
-    sk = Železničná stanica
-    sv = Tågstation
-    th = สถานีรถไฟ
-    tr = Tren İstasyonu
-    uk = Залізничний вокзал
-    vi = Đường tàu hỏa
-    zh-Hans = 火车站建筑
-    zh-Hant = 鐵道
+    en = Station Building
+    af = Stasiegebou
+    ar = مبنى المحطة
+    az = Stansiya Binası
+    be = Будынак вакзала
+    bg = Сграда на гарата
+    ca = Edifici de l'Estació
+    cs = Staniční budova
+    da = Stationsbygning
+    de = Bahnhofsgebäude
+    el = Κτίριο σταθμού
+    es = Edificio de la estación
+    et = Jaamahoone
+    eu = Geltoki Eraikina
+    fa = ساختمان ایستگاه
+    fi = Asemarakennus
+    fr = Bâtiment de la gare
+    he = בניין התחנה
+    hi = स्टेशन भवन
+    hu = Állomásépület
+    id = Bangunan Stasiun
+    it = Edificio della stazione
+    ja = 駅ビル
+    ko = 역 빌딩
+    lt = Stoties pastatas
+    mr = स्टेशन बिल्डिंग
+    nb = Stasjonsbygningen
+    nl = Stationsgebouw
+    pl = Budynek stacji
+    pt = Edifício da estação
+    ro = Clădirea gării
```

Comment:
Not sure why this change was made, "Station Building" doesn't sound natural to me, but then I'm not a native English speaker. Anyway, for Romanian the old translation was clunky, but the new one is not much better, I suggest simply:
```suggestion
    ro = Gară
```

Reply:
> the new translation is not for the Train Station POI (railway=station), but for the building usage (building=train_station)

That seems like an extremely subtle distinction, I wouldn't know when to use `building=train_station`. But anyway, in this case `Clădirea gării` is fine for Romanian.

Reply:
updated the railway.station translation to Gară, thanks 👍️